### PR TITLE
Issue341

### PR DESCRIPTION
**Pull request by babybunny**
check it out.   Adding a way to associate captains with sites. 

This is more FYI but I wanted to start getting into the rhythm of passing changes under your noses for review.  Please point out areas for improvement or ask any questions you have about how things work.

-L

### CODE REVIEW

File: room/wsgi_service.py [hunk 1]
```diff
@@ -881,6 +881,35 @@ class Retrieval(messages.Message):
   notes = messages.StringField(6)
```
Comment (andy-d) [inline room/wsgi_service.py:883]: Pre-existing issue but these are unused imports from what I can tell
import os
from google.appengine.api import users

**Comment by kate-mills** [comment]
Luke, this looks great!  Thanks for sharing.

File: app/js/rooms.js [hunk 2]
```diff
@@ -21,7 +21,8 @@ require(
             } else {
```
Comment (andy-d) [inline app/js/rooms.js:21]: lines 8, 9, 15, 65 there's a hanging comma which apparently can cause issues with javascript engine implementations (according to eslint)

**Comment by babybunny** [comment]
thanks.  will remove the unused imports.

**Pull request merged**